maven-plugin/src/main/java/org/openmrs/maven/plugins/Reset.java

```diff
@@ -63,7 +63,6 @@ public void executeTask() throws MojoExecutionException, MojoFailureException {
                 }
             }
         }
-        boolean isPlatform = StringUtils.isBlank(server.getVersion());
```

Comment:
This would always be false, due to DistroProperties constructor providing the value `1.0` (which appears inexplicable to me) as the server version initialization value.